### ISSUE TITLE
The `getOptionLabel` method of Autocomplete returned undefined instead of a string for ""

### ISSUE THREAD

**Issue by TwoForTen** [gerhat/material-ui-formik-components#32]
After frustrating 2+ hours of trying to get Autocomplete to work, I literally copied the example code and it still doesn't work.

The error is in the title. How is this possible...

**Comment by TwoForTen** [comment]
`<FastField
                        name="country"
                        options={[
                          { value: '', label: '-- No selection --' },
                          { value: 'Male', label: 'Male' },
                          { value: 'Female', label: 'Female' },
                          { value: 'Other', label: 'Other' },
                        ]}
                        component={Autocomplete}
                        textFieldProps={{
                          label: 'Država',
                          variant: 'outlined',
                        }}
                      />`

This is the Autocomplete part of the code. It is wrapped in a standard Formik form

**Comment by TwoForTen** [comment]
<FastField
                        name="country"
                        options={[
                          { value: '', label: '-- No selection --' },
                          { value: 'Male', label: 'Male' },
                          { value: 'Female', label: 'Female' },
                          { value: 'Other', label: 'Other' },
                        ]}
                        component={Autocomplete}
                        textFieldProps={{
                          label: 'Država',
                          variant: 'outlined',
                        }}
                      />

This is the autocomplete code, it is wrapped in a standard Formik form

**Comment by gerhat** [comment]
@TwoForTen thanks for pointing it out. I forgot to update the example in `codesandbox`.
In `initialValues`, country was an empty array, while it should be `null`. See below:

**Wrong:**
```javascript
initialValues={{
  username: "",
  gender: "Female",
  country: ""
}}
```

**Correct:**
```javascript
initialValues={{
  username: "",
  gender: "Female",
  country: null
}}
```

You can always refer to the repo's [example folder](https://github.com/gerhat/material-ui-formik-components/tree/master/example) for most accurate examples.

**Comment by TwoForTen** [comment]
Hey man, thank you very much for your response.
That actually got rid of the error, and the form behaves properly.

Now the next problem I have is when submitting form I get an error from the backend that the input of the Country is not a string.

Do you know anything that might be wrong from the top of your head?

**Comment by gerhat** [comment]
Are your options in this format?

```javascript
[
  {
    "label": "Label1",
    "value": "Val1"
  },
  {
    "label": "Label2",
    "value": "Val2"
  },
]
```

**Comment by TwoForTen** [comment]
Yes, the idea is that there are some options that the user can choose from, but if none match, he should be able to type in a custom value.

Judging by your answer, I reckon there's no support for it :(

**Comment by gerhat** [comment]
Correct, it's not supported yet. Do you want me to check it?

**Comment by TwoForTen** [comment]
Well I don't know how much of a hassle it is, but I will definitely try to hack my way through 😄
